src/main/java/org/jenkinsci/infra/log/ListOfPublicPlugins.java

```diff
@@ -20,9 +20,8 @@
     public ListOfPublicPlugins(Scrambler scrambler) throws IOException {
         this.scrambler = scrambler;
 
-        String s = IOUtils.toString(new URL("http://updates.jenkins-ci.org/update-center.json").openStream(), "UTF-8");
-        s = s.substring(s.indexOf('{'));
-        s = s.substring(0, s.lastIndexOf('}')+1);
+        // use the experimental update center to include plugins that are considered alpha/beta only
+        String s = IOUtils.toString(new URL("http://updates.jenkins-ci.org/experimental/update-center.actual.json").openStream(), "UTF-8");
```

Comment:
OK, I was unsure and checked, that UC does indeed contain everything, not just [alphas and betas](https://www.youtube.com/watch?v=N-aK6JnyFmk). So \o/